chainer/functions/__init__.py

```diff
@@ -68,6 +68,8 @@
 from chainer.functions.array.resize_images import ResizeImages  # NOQA
 from chainer.functions.array.rollaxis import rollaxis  # NOQA
 from chainer.functions.array.rollaxis import Rollaxis  # NOQA
+from chainer.functions.array.scatter_add import scatter_add  # NOQA
+from chainer.functions.array.scatter_add import ScatterAdd  # NOQA
```

Comment:
Could you remove `Function` class alias (capital one) for now?
We're currently reconsidering whether class aliases should be made public.

Reply:
OK